pyproject.toml

```diff
@@ -0,0 +1,82 @@
+[build-system]
+requires = ["poetry-core>=1.0.0", "poetry-dynamic-versioning>=1.0.0,<2.0.0"]
+build-backend = "poetry_dynamic_versioning.backend"
+
+[tool.poetry-dynamic-versioning]
+enable = true
+strict = false
+vcs = "git"
+style = "pep440"
+dirty = true
+
+[tool.poetry]
+name = "readlif"
+# This version is a placeholder. It is updated at build time by poetry-dynamic-versioning.
+version = "0.0.0"
```

Comment:
Should this be 0.6.5?

Reply:
This variable is dynamically controlled by poetry-dynamic-versioning, which by convention is 0.0.0.

But on this topic @keithchev, `__version__` in `__init__.py` should be replaced with 0.0.0 as well.

Reply:
Ahh what a sensible comment...